rotkehlchen/chain/ethereum/interfaces/ammswap/ammswap.py

```diff
@@ -994,6 +994,25 @@ def _get_balances_graph(
         )
         return protocol_balance
 
+    def add_lp_tokens_to_db(
+            self,
+            lp_balances_mappings: AddressToLPBalances,
+            protocol: Literal['uniswap', 'sushiswap'],
```

Comment:
the protocol should absolutely contain the version. In fact it should be the same as what we use in each module's constants. For example for uniswap v1 it's the `CPT_UNISWAP_V1 = 'uniswap-v1'` constant. For v2 make a similar one as a constant and use it.

Same for sushiswap etc.